shell/platform/embedder/embedder_external_view.cc

```diff
@@ -95,11 +101,15 @@ bool EmbedderExternalView::Render(const EmbedderRenderTarget& render_target) {
   if (!canvas) {
     return false;
   }
-
-  canvas->setMatrix(surface_transformation_);
-  canvas->clear(SK_ColorTRANSPARENT);
-  canvas->drawPicture(picture);
-  canvas->flush();
+  auto dl_canvas = std::make_unique<DlSkCanvasAdapter>(canvas);
+  int restore_count = dl_canvas->GetSaveCount();
+  dl_canvas->SetTransform(surface_transformation_);
+  dl_canvas->Clear(SK_ColorTRANSPARENT);
```

Comment:
Technically, while SK_Color constants are automatically interchangeable with DlColor instances (they both just boil down to a uint32_t), It would be better to use `DlColor::kTransparent()` here to continue the theme of non-skia use while talking to Dl objects.